classes/EventRegistry.php

```diff
@@ -263,4 +265,77 @@ public function pruneTranslatedContentTemplates($templates)
             return !Str::endsWith($template->getBaseFileName(), $extensions);
         });
     }
+
+    /**
+     * Adds language suffixes to mail view files.
+     * @param  \October\Rain\Mail\Mailer $mailer
+     * @param  \Illuminate\Mail\Message $message
+     * @param  string $view
+     * @param  array $data
+     * @param  string $raw
+     * @param  string $plain
+     * @return string|null
```

Comment:
```suggestion
     * @return bool|void Will return false if the translation process successfully replaced the original message with a translated version to prevent the original version from being processed.
```